### DISCUSSION TITLE
Wrapping up June with Gratitude 🥲 [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#59675] [New to GitHub]
I’m amazed at the awesome feedback, thoughtful questions asked, and members helping each other out this past month and want to take a moment to express a moment of gratitude for all y’all do to make New to GitHub the place to be ❣️. Some folks in particular I want to shout out and thank as well… 

🌟**Community Superstars**: A round of applause for these community citizens 👏 
* @arzuunrk: They included a ton of detail in [their post](https://github.com/orgs/community/discussions/58410) including what they already tried to rectify an authentication error. They still need help so go see if you can assist and save the day 🦸
* @Mike1794: For asking a curious question about assigning tasks to users not on GitHub Enterprise - [check it out](https://github.com/orgs/community/discussions/58254). 
* @RdKaiC: Created one of our top [posts](https://github.com/orgs/community/discussions/51727) about removing payment methods that many of you weighed in on - we passed this feedback on internally and are awaiting more information. Thank you!

🆒 **Cool stuff**™️ happening around GitHub and the Community
* **Happy Pride** 🌈 🏳️‍🌈 🏳️‍⚧️ ❣️: Celebrate Pride with GitHub and the community. [View our post](https://github.com/orgs/community/discussions/58667) on queer open source projects to contribute to and more.
* **Large Language Models Use Cases Discussion** 🧠: [Join the conversation](https://github.com/orgs/community/discussions/59354) on how you use LLMs with @sethjuarez. 
* **Top Reasons NOT to use GitHub Copilot** 🧑‍✈️: Are you considering using Copilot? @colindembovsky helps you decide with his [latest post](https://github.com/orgs/community/discussions/58562). 

📚 **Rousing Reads and Beyond**: GitHub-produced goodies 🍭
* **Bridging code and community** 🌉: Explore the impact of non-code contributions—and why they are often undervalued, the challenges of using open source in regulated environments, and the art of managing projects at the scale of Kubernetes, now on [The ReadME Podcast](https://github.blog/2023-06-21-bridging-code-and-community/). Then join [our discussion](https://github.com/orgs/community/discussions/58571) to let us know what you think 💬.

Have any open source projects that you contribute to that help the LGBTIA+ community? Link them below to round out Pride month 🌈.

**Comment by Kashumy** [comment]
-_- ?

**Comment by queenofcorgis** [comment]
Thanks to everyone who helped us start July hot - check out some spotlighted contributors and what else is new in our [newest check-in](https://github.com/orgs/community/discussions/60797).
